contribution-and-developer-guidelines.md

```diff
@@ -0,0 +1,102 @@
+# CoatRack Contribution and Developer Guidelines
```

Comment:
As developers are a sub-group of contributors, I think the following should be sufficient:
```suggestion
# CoatRack Contributor Guidelines
```

Besides, I would suggest to rename this file as `CONTRIBUTING.md`, as that is the filename recommended in the FIWARE requirements. I would also suggest to include the "signing instructions" in this file, if there is no obvious reason to do otherwise.

Reply:
Done. I merged the guidelines and the signing instructions into a CONTRIBUTING.md file.

Reply:
Just a minor note: the ODT/PDF files still refer to "signing-template.md", should be changed.